test.yaml

```diff
@@ -0,0 +1,62 @@
+[?1049h[>4;2m[?1h=[?2004h[1;62r[?12h[?12l[22;2t[22;1t[27m[29m[m[H[2J[?25l[62;1H"/var/folders/dl/cj09tpkx5tl8dh37_f3_b0lr0000gn/T/kubectl-edit-61t0l.yaml" 67L, 1624B[1;1H[4m[93m  1 [m[4m[96m# Please edit the object below. Lines beginning with a '#' will be ignored,[m[4m                                                                                                                                                               [m[2;1H[93m  2 [m[96m# and an empty file will abort the edit. If an error occurs while saving this file will be[m

```

Comment:
How do we use this file

Reply:
It's just for testing. I will delete it.